editors/vim/syntax/lambdapi.vim

```diff
@@ -27,26 +27,32 @@ highlight link StringLiteral String
 
 " Keywords.
```

Comment:
Idem, could you please put them in alphabetical order?

Reply:
fixed in 24c13d9